testing/src/cypress/e2e/04-api-institution/01-instition-dashboard.cy.ts

```diff
@@ -0,0 +1,150 @@
+const COLLEGE_F = Cypress.env("COLL_F");
+const USER_F = Cypress.env("USERS").coll_f;
+const BASE_URL = Cypress.env("TEST").BASE_URL + "/api/institutions";
+const TOKEN = Cypress.env("TEST").TOKEN;
+
+describe("Institution apis", () => {
+  it("Get user status", () => {
+    cy.request({
+      method: "GET",
+      url: `${BASE_URL}/institution-user/status`,
+      followRedirect: false,
+      headers: {
+        "Content-Type": "text/html",
+        Authorization: TOKEN,
+      },
+    }).then((response) => {
+      expect(response.status).to.be.equal(200);
+      expect(response.body).to.have.property("isExistingUser", true);
+      expect(response.body).to.have.property("isActiveUser", true);
+    });
+  });
+
+  it("Get user details", () => {
+    cy.request({
+      method: "GET",
+      url: `${BASE_URL}/institution-user/my-details`,
+      followRedirect: false,
+      headers: {
+        "Content-Type": "text/html",
+        Authorization: TOKEN,
+      },
+    }).then((resp) => {
+      expect(resp.status).to.be.equal(200);
+      expect(resp.body.user).to.have.property("userName", USER_F.userName);
+      expect(resp.body.user).to.have.property("firstName", USER_F.firstName);
+      expect(resp.body.user).to.have.property("lastName", USER_F.lastName);
+      expect(resp.body.user).to.have.property("isActive", true);
+      expect(resp.body.user).to.have.property(
+        "userFullName",
+        USER_F.userFullName
+      );
+      expect(resp.body.user).to.have.property("isAdmin", true);
+      expect(resp.body.user).to.have.property("email", USER_F.email);
+      expect(resp.body.authorizations).to.have.property(
+        "institutionId",
+        USER_F.authorizations.institutionId
+      );
+      expect(resp.body.authorizations.authorizations[0]).to.have.property(
+        "locationId",
+        USER_F.authorizations.authorizations[0].locationId
```

Comment:
what happens here when the logged-in institution user does to have a location? won't it throw an error?

Reply:
the value for the location would be null, and this is handled in the expected results. So the user's authorizations are defined accordingly in a separate file.

Reply:
As discussed will add a separate test case for the scenario